### DISCUSSION TITLE
Export API interface

### DISCUSSION THREAD

**Discussion by duty9527** [yarox24/EvtxHussar#5] [Ideas]
The problem I am currently facing is that I need to parse the logs of the Windows system and send them to the server. In this case, if there are relevant API interfaces available, there is no need to delve into the source code understanding.

**Comment by yarox24** [comment]
Hi blue-duty, I can add some features rather to command line as this tool is not library. I need to know more of the use case.

Should I add something like:
```
--output_remote
--remote-host SOME_IP
--remote-port SOME_PORT
--protocol <Which protocol here? S3/HTTP(S)/RAW/Elastic/other?>
```

**Reply by duty9527** [reply]
While this doesn't serve my purpose, adding features like remote sending is a good direction
